website/docs/release-notes/2023-11-15-version-200.md

```diff
@@ -0,0 +1,176 @@
+---
+id: release-02
+title: AmigaOS 2.0 Barley Release Notes
+authors: [mihelich, hackerman342, guilhermedemouraa]
+tags: [announcement, release]
+---
+
+![barley-logo](https://github.com/farm-ng/amiga-dev-kit/assets/484900/df9e214a-1dcf-4332-acbb-b73d00e2d1fc)
+
+We are excited to announce AmigaOS 2.0 “Barley,” a major release providing the foundation of
+built-in autonomy features and powerful custom application development on the Amiga robot.
+
+Highlights of Barley include:
+
+* Core capabilities for autonomy and precision navigation:
+  * Precise geo-location, up to 2cm accuracy, with integrated RTK-GPS receiver.
+  * Robust tracking of robot movement (position and orientation), fusing GPS, IMU, and wheel
+    odometry.
+* Autoplot app providing user-friendly control of the Amiga, including:
+  * Teach-and-repeat workflow for automating navigation tasks.
+  * Remote teleoperation from any web browser.
+* Developer-focused environment for custom application development, including:
+  * Ubuntu 20.04 and Jetpack 5.1 for NVIDIA Jetson ecosystem compatibility.
+  * Remote SSH access from any network.
+  * API access to all sensor data streams, robot state estimation and control, and autonomous track
+    following.
+  * Custom web application deployment to the Brain display.
+  * Optional ROS bridge.
+* Bandwidth-optimized over-the-air (OTA) updates for future AmigaOS releases.
+
+:::warning Upgrading from 1.0
+
+Due to the significant platform- and OS-level changes, existing Brains running AmigaOS 1.0 require a
+one-time physical upgrade by farm-ng staff to install the GPS receiver module and re-image the Brain
+to AmigaOS 2.0. Please contact support@farm-ng.com if your Brain has not been upgraded already.
+Future software updates will be provided over-the-air.
+
```

Comment:
I think it's also important to clarify that this OS doesn't have backward compatibility. I.e., apps developed in the OS 1.0 won't work in the OS 2.0